packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/selectedClass.st

```diff
@@ -0,0 +1,12 @@
+code browser
+selectedClass
+
+	self currentSelection 
+	ifNotNil: [
+		(self checkIfClassExists: self currentSelection )
+		ifTrue: 
+			[^ self currentSelection method methodClass theNonMetaClass]
+	]. 
```

Comment:
It is a Squeak best-practice to close braces on the last line of the wrapped block. So in your case `}.` would be the last two characters on line 8.